Content/tutorials/tuist-xcode-project-setup.md

```
@@ -0,0 +1,395 @@
+
+```
+mise exec tuist -- tuist generate
+```
+
+This will create the Xcode project and workspace where you can work on your app.
+
+**Remember don't edit the project and workspace setting in Xcode directly. Edit the Project.swift file and other tuist related Swift files to save your changes to the repo.**
+
+Now that we've set up our first project using Tuist, let's dive into how the Xcode project works. The generated `Project.swift` uses `buildableFolders`, but for the walkthrough below we'll use the simpler `sources` parameter — it's easier to reason about as we build up to the final version.
+
+### Projects and their Targets
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)